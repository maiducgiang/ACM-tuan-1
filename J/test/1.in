5
84,56(50)
27,74(59)
13,14(52)
7,77(80)
46,49(51)
