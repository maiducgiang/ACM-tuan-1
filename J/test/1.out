418597 4950
54937 1980
65069 4950
77003 9900
76717 1650
